Library/Homebrew/cli/parser.rb

```diff
@@ -432,6 +443,77 @@ def hide_from_man_page!
 
       private
 
+      SYMBOL_TO_USAGE_MAPPING = {
+        text_or_regex: "<text>|`/`<regex>`/`",
+        url:           "<URL>",
+      }.freeze
+
+      def generate_usage_banner
+        command_names = ["`#{@command_name.tr("_", "-")}`"]
```

Comment:
Can the `.tr("_", "-")` be moved to where `@command_name` is defined?

Reply:
`HOMEBREW_INTERNAL_COMMAND_ALIASES` actually include the names with hyphens, to the `map` below` will not work for e.g. `--repository`.

Reply:
Good call. Thanks, @reitermarkus!